api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -71,8 +115,169 @@ This endpoint returns a list of aggregated award amounts grouped by time period
             + `month`
         + results (array[TimeResult], fixed-type)
 
+## Download Count [/api/v2/download/count/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Download Count Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (required, object)
+            + time_period (array[FilterObject])
+
++ Response 200 (application/json)
+    + Attributes
+        + transaction_rows_gt_limit : true (required, boolean)
+            A boolean returning whether the transaction count is over the maximum row limit.
+
+
+## Generated Filter Hash for URL [/api/v1/references/filter/]
+
+Generates a hash for URL, based on selected filter criteria.
+
+### Generated Filter Hash for URL Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (optional, object)
+
++ Response 200 (application/json)
+    + Attributes
+        + hash : `96982f90346b1360dc5fb0a97d4b23fa` (required, string)
+
+
+## Restore Filters From URL Hash [/api/v1/references/hash/]
+
+Restores selected filter criteria, based on URL hash.
+
+### Restore Filters From URL Hash Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + hash : `96982f90346b1360dc5fb0a97d4b23fa` (required, string)
+
++ Response 200 (application/json)
+    + Attributes
+        + filter (optional, object)
+            + filters (optional, object)
+                + recipientDomesticForeign : `all` (required, string)
+                + selectedFundingAgencies : (required, object)
+                + selectedPSC : (required, object)
+                + awardAmounts : (required, object)
+                + selectedNAICS : (required, object)
+                + timePeriodFY : `2019` (required, array[string])
+                + selectedRecipients : (required, array[string])
+                + recipientType : (required, array[string])
+                + timePeriodEnd : (required, string)
+                + selectedRecipientLocations : (required, object)
+                + timePeriodStart : (required, string)
+                + locationDomesticForeign : `all` (required, string)
+                + extentCompeted : (required, array[string])
+                + selectedAwardingAgencies : (required, object)
+                + setAside : (required, array[string])
+                + pricingType : (required, array[string])
+                + awardType : (required, array[string])
+                + timePeriodType : `fy` (required, string)
+                + selectedCFDA : (required, object)
+                + keyword : (required, object)
+                + selectedAwardIDs : (required, object)
+                + selectedLocations : (required, object)
+            + version: `2017-11-21` (optional, string)
+
 # Data Structures
 
+# FilterObjectAwardTypes (array)
```

Comment:
This should be a level 2 heading (`##`) because it falls under `# Data Structures`